mozilla-release/toolkit/modules/NewTabUtils.jsm

```diff
@@ -11,16 +11,18 @@ ChromeUtils.import("resource://gre/modules/XPCOMUtils.jsm");
 Cu.importGlobalProperties(["btoa", "URL"]);
 
 // Android tests don't import these properly, so guard against that
+let didSuccessfulImport = false;
+#if 0
 let shortURL = {};
 let searchShortcuts = {};
-let didSuccessfulImport = false;
```

Comment:
lets not delete it to prevent conflict, it will not execute as its under if 0

Reply:
I moved it higher